opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/controller.py

```diff
@@ -26,18 +26,18 @@ class PushController(threading.Thread):
     exports them and performs some post-processing.
 
     Args:
-        meter: The meter used to collect metrics.
+        accumulator: The meter used to collect metrics.
```

Comment:
please update the description. I believe it's incorrect that any meter from any SDK will work in this architecture.

Reply:
Is this more accurate?
`accumulator: The meter from the Metrics API used to collect metrics.`